spacer/extract_features.py

```diff
@@ -111,12 +114,50 @@ def feature_dim(self):
 
 class EfficientNetExtractor(FeatureExtractor):
 
+    def __init__(self):
+
+        # Cache models locally.
+        self.modelweighs_path, self.model_was_cashed = download_model(
+            'efficientnetb0_5eps_best.pt')
+
     def __call__(self, im, rowcols):
-        raise NotImplementedError
+
+        start_time = time.time()
+
+        # Set torch parameters
+        torch_params = {'model_type': 'efficientnet',
+                        'model_name': 'efficientnet-b0',
+                        'weights_path': self.modelweighs_path,
+                        'num_class': 1279,
+                        'crop_size': 224,
+                        'batch_size': 10}
+
+        # The imdict data structure needs a label, set to 1, it's not used.
+        rowcollabels = [(row, col, 1) for row, col in rowcols]
```

Comment:
Why do you use the label here? Maybe it's time to clean that up? It's some old legacy thing I never got around to removing.  Also, please re-factor the caffe extractor to use the same shared utils.

Reply:
label arg removed since it is not needed so I re-factored the Caffe extractor as well